drafts/draft-0002.md

```diff
@@ -0,0 +1,118 @@
+3. `pubkey` (optional) - Specify which public key you want the URI handler to sign for. Useful with the `submit` parameter above for example with multisig coordination.
+4. `msg` (optional) -  There can be an optional `msg` query param to indicate any additional information that the website or application wants to show the user in her wallet. The value for this query param should be URL-encoded as well and should not be longer than 300 characters before the URL-encoding. Note that the `msg` field is _different_ from the `memo` field that is included in a transaction. The `msg` field will not be put on-chain, but the `memo` field will be put on-chain.
+
+**Example 1 - Payment Operation with source account needing replacement**:
+
+`stellar://tx/?xdr=AAAAAL6Qe0ushP7lzogR2y3vyb8LKiorvD1U2KIlfs1wRBliAAAAZAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAABEz4bSpWmsmrXcIVAkY2hM3VdeCBJse56M18LaGzHQUAAAAAAAAAAACadvgAAAAAAAAAAA`
+
+**Example 2 - Change Trust Operation with submit endpoint specified**:
+
+`stellar://tx/?xdr=AAAAAP%2Byw%2BZEuNg533pUmwlYxfrq6%2FBoMJqiJ8vuQhf6rHWmAAAAZAB8NHAAAAABAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAA%2F7LD5kS42DnfelSbCVjF%2Burr8GgwmqIny%2B5CF%2FqsdaYAAAAAAAAAAACYloAAAAAAAAAAAA&submit=url%3Ahttps%3A%2F%2FsomeSigningService.com%2Fa8f7asdfkjha&pubkey=GAU2ZSYYEYO5S5ZQSMMUENJ2TANY4FPXYGGIMU6GMGKTNVDG5QYFW6JS&msg=order%20number%2024`
+
+### Operation `pay`
+The `pay` operation represents a request to pay a specific address with a specific asset, regardless of the source asset used by the payer. If the payer decides to use a different source asset then the wallet should leverage the [path payment operation](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#path-payment) to achieve this. The parameters for the `pay` operation are as follows:
+1. `destination` (required) - A valid account ID or payment address
+2. `amount` (required) - Amount that destination will receive
+3. `asset_code` (optional) - Asset code (XLM if not present) destination will receive
+4. `asset_issuer` (optional) - Account ID of asset issuer (XLM if not present) destination will receive
+5. `memo` (optional) - Can be a memo to be included in the payment / path payment. This should follow the normal guidelines for memos. This should also be URL-encoded.
```

Comment:
We should also have a param for `memo_type` and agree on encoding of `memo_hash`: hex or base64?